net.zdechov.app.C-evo.yml

```diff
@@ -0,0 +1,58 @@
+app-id: net.zdechov.app.C-evo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
```

Comment:
the freedpascal SDK is not available for 24.08 so it would be better to use that.

```suggestion
runtime-version: '24.08'
```

Reply:
That looks like contradiction. I habe also tested 24.08 and came to the same conclusion. Freepascal SDK is not yet available so newer runtime can't be used yet. Then there is no action here.

Reply:
it's now available for 24.08, that's a typo

Reply:
yeah I'm the king of typos. When it's not the autocarrot.